examples/quickstart-xgboost/requirements.txt

```diff
@@ -0,0 +1,4 @@
+flwr>=1.0, <2.0
+flwr_datasets
```

Comment:
It will need to be 0.0.2 or higher both here and in pyproject.toml. Also let's make the versions' specifications exactly the same both in pyproject.toml and requirements.txt